docs/en/config_fw/gain_compression.md

```diff
@@ -0,0 +1,18 @@
+# Gain compression
```

Comment:
1. Is this topic expected to be "Gain Compression Overview" or "Gain Compression Tuning". At the moment there it is almost all overview and no guidance.
2. How should I set FW_GC_GAIN_MIN? What if I get it wrong?
3. Do I disable it for autotune?

If this is just an overview of the feature, then you might put this under "Features" - no point describing a tuning step where the user needs to do nothing.

Make sense?

Reply:
done in 18abd26b0190b4bca464c652bdd6fdaf5dabd92d